src/org/labkey/test/components/glassLibrary/cards/SampleTypeCard.java

```diff
@@ -0,0 +1,122 @@
+package org.labkey.test.components.glassLibrary.cards;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.Component;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import static org.labkey.test.components.html.Input.Input;
+
+public class SampleTypeCard extends WebDriverComponent<SampleTypeCard.ElementCache>
```

Comment:
Include a link to the product component instrumented by this class.

Reply:
done